docs/static/releasenotes.asciidoc

```diff
@@ -15,6 +16,15 @@ See also:
 * <<release-notes-xls>>
 endif::include-xpack[]
 
+[[logstash-6.0.0-rc2]]
+=== Logstash 6.0.0-rc2 Release Notes
+
+* https://github.com/elastic/logstash/commit/840439722d8ef4737c7e8101c59652ced191bbea[Improved performance]
+* Fixed bug where logstash would crash when logging using the `debug` verbosity level
```

Comment:
I'm not sure if this was an issue for RC1, just 5.6.3  - does it matter?

Reply:
You are right! Will remove

Reply:
Still present... ;)